Francisco_Trujillo/Asignments/Solar/solar_system_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative 'solar_system'
+RSpec.describe Planet do
+	before do
+		@earth = Planet.new
+	end
+	it 'check for the class' do
+		expect1 =''
+		expect2 = 0
+		expect(@earth.name).to eq(expect1)
+		expect(@earth.description).to eq(expect1)
+		expect(@earth.population).to eq(expect2)
```

Comment:
It would be better to initialize a planet with a name, description and population and then use the tests to make sure it's initializing correctly.  If you just want to check if the class is correct then it should be expect(@earth.class.ancestors.include?(Planet)).to eq(true)